src/pylast/__init__.py

```diff
@@ -329,14 +328,14 @@ def get_top_tracks(self, limit=None, cacheable=True):
 
         return seq
 
-    def get_top_tags(self, limit=None, cacheable=True):
+    def get_top_tags(self, limit=None, cacheable: bool = True):
         """Returns the most used tags as a sequence of TopItem objects."""
 
         # Last.fm has no "limit" parameter for tag.getTopTags
         # so we need to get all (250) and then limit locally
         doc = _Request(self, "tag.getTopTags").execute(cacheable)
 
-        seq = []
+        seq: list[TopItem] = []
```

Comment:
Subscripting classes like `list`  is only supported from Python 3.9 onwards. Is dropping support for earlier versions of Python intentional?

Reply:
No, I want to keep support for 3.7+ (following CPython EOLs).

Doesn't `from __future__ import annotations` add support for 3.7? Tests are passing for 3.7-3.10.

Reply:
Interesting, I guess it works because the type annotation is lazily evaluated. I wonder if code that attempts to analyse those type annotations would still work though.